fhir-bulkimportexport/src/main/java/com/ibm/waston/health/fhir/bulkcommon/COSUtils.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.waston.health.fhir.bulkcommon;
+
+import java.io.InputStream;
+import java.util.List;
+import java.util.logging.Logger;
+
+import com.ibm.cloud.objectstorage.ClientConfiguration;
+import com.ibm.cloud.objectstorage.SDKGlobalConfiguration;
+import com.ibm.cloud.objectstorage.auth.AWSCredentials;
+import com.ibm.cloud.objectstorage.auth.AWSStaticCredentialsProvider;
+import com.ibm.cloud.objectstorage.auth.BasicAWSCredentials;
+import com.ibm.cloud.objectstorage.client.builder.AwsClientBuilder.EndpointConfiguration;
+import com.ibm.cloud.objectstorage.oauth.BasicIBMOAuthCredentials;
+import com.ibm.cloud.objectstorage.services.s3.AmazonS3;
+import com.ibm.cloud.objectstorage.services.s3.AmazonS3ClientBuilder;
+import com.ibm.cloud.objectstorage.services.s3.model.AbortMultipartUploadRequest;
+import com.ibm.cloud.objectstorage.services.s3.model.Bucket;
+import com.ibm.cloud.objectstorage.services.s3.model.CompleteMultipartUploadRequest;
+import com.ibm.cloud.objectstorage.services.s3.model.InitiateMultipartUploadRequest;
+import com.ibm.cloud.objectstorage.services.s3.model.InitiateMultipartUploadResult;
+import com.ibm.cloud.objectstorage.services.s3.model.PartETag;
+import com.ibm.cloud.objectstorage.services.s3.model.UploadPartRequest;
+import com.ibm.cloud.objectstorage.services.s3.model.UploadPartResult;
+
+/**
+ * Utility functions for IBM COS.
+ * 
+ * @author Albert Wang
+ */
+
+public class COSUtils {
+    private final static Logger logger = Logger.getLogger(COSUtils.class.getName());
+
+    /**
+     * Logging helper.
+     */
+    private static void log(String method, Object msg) {
+        logger.info(method + ": " + String.valueOf(msg));
+    }
+    
+    public static String startPartUpload(AmazonS3 cosClient, String bucketName, String itemName) throws Exception {
+        try {
+            log("startPartUpload", "Start multi-part upload for " + itemName + " to bucket:" + bucketName);
+    
+            InitiateMultipartUploadResult mpResult = cosClient.initiateMultipartUpload(new InitiateMultipartUploadRequest(bucketName, itemName));
+            return mpResult.getUploadId();
+        } catch (Exception sdke) {
+            log("startPartUpload", "Upload start Error: " + sdke.getMessage());
+            throw sdke;
+        }
+    }
+    
+    
+    public static AmazonS3 getCosClient(String cosCredentialIbm, String cosApiKeyProperty, String cosSrvinstId, 
+            String cosEndpintUrl, String cosLocation) {
+        SDKGlobalConfiguration.IAM_ENDPOINT = "https://iam.cloud.ibm.com/oidc/token";
```

Comment:
probably a comment for last PR rather than this one, but use of ALL CAPS here makes it look like a constant when apparently its not

Reply:
albert pointed out that this comes from the COS SDK, not from us

Reply:
this constant is defined by COS SDK.